dataactbroker/README.md

```diff
@@ -616,14 +618,16 @@ Example output:
           "error_name": "rule_failed",
           "field_name": "award_financial",
           "occurrences": "11",
-          "rule_failed": "fain, uri"
+          "rule_failed": "fain, uri",
+          "original_label":""
```

Comment:
Thanks for the context. Do we have a canonical list of validations we've implemented that aren't explicitly expressed in the practices and procedures doc? Since we've started the weekly check-ins, I've been trying to log things there. 

Unrelated to this particular PR, but I want to make sure we're not doing validations that aren't (or won't be) explicitly written out by the schema team, since those docs are what agencies will use to prep their data.